22/jdk/alpine/3.19/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.19
+
+ARG version=22.0.0.36-2
```

Comment:
The version here should be 22.0.0.36.2, not sure why this is different from previous versions yet.